tests/unit/sequential/test_par.py

```diff
@@ -32,7 +32,7 @@ def get_metadata(self, add_sequence_key=True, add_sequence_index=False):
 
     def get_data(self):
         data = pd.DataFrame({
-            'time': [1, 2, 3],
```

Comment:
I actually use datetimes as now `auto_assign_transformer` will validate this data which would error out. It should be date-time anyway